op-supervisor/supervisor/backend/backend.go

```diff
@@ -176,10 +196,52 @@ func NewSupervisorBackend(ctx context.Context, logger log.Logger,
 func (su *SupervisorBackend) OnEvent(ev event.Event) bool {
 	switch x := ev.(type) {
 	case superevents.LocalUnsafeReceivedEvent:
+		// Check if the block is in an interop active period for this chain
```

Comment:
Let's check that unsafe blocks are a good trigger for protocol upgrades. Maybe safe is the better trigger, since it is *safe*.

Reply:
I think we landed on using Finalized blocks.